testing/rspamd/APKBUILD

```diff
@@ -35,6 +35,7 @@ build() {
 		-DRSPAMD_USER=$pkgusers \
 		-DRSPAMD_GROUP=$pkggroups \
 		-DENABLE_HIREDIS=ON \
+		-DENABLE_LUAJIT=OFF \
```

Comment:
This is not a binary option, it’s possible to disable specific PaX protections for specific binaries. We do it even for nginx.